hunt/app/views/animals/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<ul>
+  <% @animals.each do |animal| %>
+    <!--
+      <li>
+        <% link_to animal.name, animal_path(animal)  %></li>
+      -->
+    <div class="container">
+      <div class="row">
+        <div class="col-sm">
+          <%='Name: '+animal.name %>
```

Comment:
Instead of using string concatenation on these, you could do `Name: <%= animal.name %>`